core/src/validator.rs

```diff
@@ -924,6 +928,7 @@ impl Validator {
             last_full_snapshot_slot,
             block_metadata_notifier,
             config.wait_to_vote_slot,
+            no_serve_snapshots,
```

Comment:
just use `config.no_serve_snapshots` here instead of creating a local?

Reply:
Agreed.. I was trying to follow the way the other values worked, but it seemed silly and I wasn't sure about that style.